lnrpc/walletrpc/walletkit.proto

```diff
@@ -400,11 +400,12 @@ message PendingSweep {
     uint32 amount_sat = 3;
 
     /*
+    Deprecated.
```

Comment:
nit: standardize deprecation notice on its own line vs inline with the rest of the comment?